bot.py

```diff
@@ -351,35 +351,48 @@ def set_and_check_team(self, player, team):
 				status = params[1]
 				if self.own_pokemon(pokemon_data):
 					self.add_status(self.statuses, pokemon_name, status)
```

Comment:
It would probably be more concise to just move calls of self.gs.set_status to self.add_status, right?

Reply:
Makes sense